### DISCUSSION TITLE
迅雷云盘无法加载

### DISCUSSION THREAD

**Discussion by Jonson-zhang** [AlistGo/alist#5212] [General]
报如下错误：
ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: meta.username expect 13611268550, but got map[phone_number:+86 13611268550 result:accept], please check captcha init params

重新删除迅雷云盘后，再添加，仍然报上面错误，请问该如何处理？

**Comment by cmj2023** [comment]
电话前面加一个+86 ，然后重新添加

**Reply by jv945** [reply]
靠谱

**Reply by huangwk0927** [reply]
兄台为何如此优秀

**Reply by xiao1521391081** [reply]
牛啊

**Reply by KennyZKY** [reply]
太秀了

**Comment by Jonson-zhang** [comment]
牛啊，解决了，谢谢

**Comment by zhouyuanh** [comment]
+86解决了
